cmake/Modules/LibAddBinding.cmake

```diff
@@ -87,6 +87,12 @@ endfunction (check_binding_included)
 #    add_binding ("anynameyouwant")
 # ~~~
 function (add_binding BINDING_NAME)
+	cmake_parse_arguments (ARG
+			       "ONLY_SHARED" # optional keywords
```

Comment:
yes since bindings don't seem to be bundled with FULL or STATIC so for those its just an info. Or should i remove it again since it doesn't really seem to matter?

Reply:
No it is okay. But aren't all bindings ONLY_SHARED currently?

Reply:
i don't know how its the case for the other bindings but for the haskell ones they only work with the shared version yes, thats why i only added it to them.